### ISSUE TITLE
deprecated message not displayed when using multiline string

### ISSUE THREAD

**Issue by Viicos** [microsoft/pyright#7319]
Code sample in [pyright playground](https://pyright-play.net/?strict=true&code=GYJw9gtgBALgngBwJYDsDmB9ApgDxllAZyTCKiQgTBBigBMsEQsBjAQ3zoCgeABBpqw5Y6ACi5RJUAEQB1agGtUaeo2bsYJFFAhZChNmizSuASi4NgUYAFcULAIyjTALigI2%2BvgPXCxEqWkAOTBaAHdFZVVBDS0dPQMjEykZWAALDnJCHRsAG01c1GMzCywrW3sAJmc3Dy8uCscGuxZKriA)

```python
from typing_extensions import deprecated


@deprecated(
    "Working deprecation message"
)
def func1(): pass

@deprecated(
    "Not working deprecation message"
    " that is multiline"
)
def func2(): pass

func1
func2
```

Hovering on `func1` correctly displays the deprecation message, however when multiline strings are used, the message isn't shown. Not sure if this is something `pyright`/Pylance can handle? Thanks in advance!

**Comment by erictraut** [comment]
A multi-line string literal in Python is written with triple quotes. 

```python
@deprecated(
"""Working deprecation message
 that is multiline"""
)
def func3():
    pass
```

Your code sample is using multiple concatenated strings. This approach isn't generally recommended, which is why it's not currently supported by pyright. Adding this support is simple, so I've gone ahead and implemented it. This will be included in the next release.

**Comment by Viicos** [comment]
Sorry I meant concatenated strings indeed. Thanks for the quick fix

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.352, which I just published.
